subiquity/common/types.py

```diff
@@ -406,7 +418,8 @@ class GuidedResizeValues:
 @attr.s(auto_attribs=True)
 class GuidedStorageTargetReformat:
     disk_id: str
-    capabilities: List[GuidedCapability]
+    allowed: List[GuidedCapability]
```

Comment:
There can be scenarios where a given Target type will not ever be allowed.  For example, imagine a disk that is too small, nothing ever will change to make that into an allowed capability.  Suggest giving it an default empty list value.

Reply:
OK